oauth2client/client.py

```diff
@@ -1872,6 +1878,7 @@ def __init__(self, client_id,
         self.device_uri = device_uri
         self.token_info_uri = token_info_uri
         self.authorization_header = authorization_header
+        self.pkce = pkce
```

Comment:
We have `DeprecationWarning` to indicate when something won't be present in a future release. But if you want this stuff to be private, that's fine by me.